conference/user_panel.py

```diff
@@ -181,7 +181,10 @@ def clean_email(self):
         return self.cleaned_data["email"]
 
     def get_user(self):
-        return User.objects.get(email=self.cleaned_data["email"])
+        return User.objects.get(
+            is_active=True,
+            email__iexact=self.cleaned_data["email"],
```

Comment:
good call, I'd probably use `email=self.cleaned_data['email'].lower()` but I'm not sure all the emails in the db are converted to lowercase (we do that on the registration from, but iirc only started doing this last year)

Reply:
Yeah, the `iexact` is probably slower, but makes no assumption about the data on the db.